products/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db                 import models
+from django.db.models.deletion import CASCADE
+
+class Category(models.Model):
+    category = models.CharField(max_length=50, unique=True)
```

Comment:
Category table의 category field는 중복성의 느낌이 있고, 나중에 관계된 데이터를 가져 올 때 헷갈릴 수 있습니다.
예) product의 category 정보를 가져올 때
- product.subcategory.category.category -> category가 중복되어 혼란을 일으킬 수 있음
- product.subcategory.category.name -> 가져 오려는 data를 명확히 할 수 있음

항상 중복성을 피해야하는 것은 아니지만, 어떻게 이름을 지었을 때 가독성 측면이나 성능 측면에서 생각하면서 지으면 좋을 것 같습니다.
다른 table들도 이 점 참고해서 수정해 주세요.
```suggestion
    name = models.CharField(max_length=50, unique=True)
```

Reply:
수정 완료했습니다!